.github/workflows/ci.yml

```diff
@@ -100,6 +91,23 @@ jobs:
           # Make sure the help options print.
           python -m PyInstaller -h
 
+      - name: Install test dependencies (tools)
+        run: |
+          pip install --progress-bar=off --upgrade --requirement tests/requirements-tools.txt
+
+      - name: Install test dependencies (libraries)
+        id: install_libraries
+        if: ${{ !endsWith(matrix.python-version, '-dev') }}
+        run: |
+          pip install --progress-bar=off --upgrade --requirement tests/requirements-libraries.txt
+
+      - name: Set up extra environment variables (non-Windows)
+        if: ${{ !startsWith(matrix.os, 'windows') }}
+        run: |
+          # Run Qt-based tests with offscreen backend. This seems to break QtWebEngine tests on Windows, so
+          # apply it only here, in non-Windows path.
```

Comment:
For what it's worth, macOS and Windows GA runners have displays (or something close enough to count as a display) attached so we could make this path Linux only.

Reply:
Probably not worth changing unless it starts to cause issues later.